pkg/serverless/binarysize/binary_size_test.go

```diff
@@ -0,0 +1,53 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package binarysize
```

Comment:
Should we remove this one as well?

Reply:
I feel like this one is still mildly helpful. While it's okay that the binary size increases now, it's still not ideal. It seems like we've determined that `github.com/h2non/filetype` is one culprit. 

Happy to hear your thoughts otherwise though! I'm all about deleting code. 😅

Reply:
Actually, I take this back. I will delete this test. This PR is _adding_ this, it wasn't there previously. We don't need it, so why add it?